From ed076173797498ade300d823ad041966eb0e1e19 Mon Sep 17 00:00:00 2001
From: Richard Russon <rich@flatcap.org>
Date: Thu, 9 Jun 2016 12:06:10 -0700
Subject: [PATCH 37/37] Fix alignment in the compose menu.

Several of the compose-panel labels didn't use the TITLE_FMT to align
themselves.  This causes formatting problems when the width is changed,
e.g. by the NNTP patch.
---
 compose.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/compose.c b/compose.c
index bf34d70..286d094 100644
--- a/compose.c
+++ b/compose.c
@@ -147,7 +147,7 @@ static void snd_entry (char *b, size_t blen, MUTTMENU *menu, int num)
 
 static void redraw_crypt_lines (HEADER *msg)
 {
-  mutt_window_mvaddstr (MuttIndexWindow, HDR_CRYPT, 0, "Security: ");
+  mutt_window_mvprintw (MuttIndexWindow, HDR_CRYPT, 0, TITLE_FMT, "Security: ");
 
   if ((WithCrypto & (APPLICATION_PGP | APPLICATION_SMIME)) == 0)
   {
@@ -213,7 +213,7 @@ static void redraw_mix_line (LIST *chain)
   char *t;
 
   /* L10N: "Mix" refers to the MixMaster chain for anonymous email */
-  mutt_window_mvaddstr (MuttIndexWindow, HDR_MIX, 0,     "     Mix: ");
+  mutt_window_mvprintw (MuttIndexWindow, HDR_MIX, 0, TITLE_FMT, _("Mix: "));
 
   if (!chain)
   {
-- 
2.9.0

